docs/source/user/api.rst

```diff
@@ -674,6 +675,26 @@ Responds only to **POST**.
 
 No parameters required.
 
+Get active SMS token status
+---------------------------
+
+.. note::
+  This API endpoint will work only if the organization has enabled
+  :ref:`SMS verification <openwisp_radius_sms_verification_enabled>`.
+
+**Requires the user auth token (Bearer Token)**.
+
+Used for SMS verification, allows users to check if a SMS code is
+already generated for their mobile number.
```

Comment:
I'd change this description to something like:

Allows checking whether an active SMS token was already requested for the mobile phone number of the logged in account.